mdx/guides/bug-bounty-program.mdx

```diff
@@ -18,22 +18,43 @@ The following are examples of types of vulnerabilities that are of interest:
 -   Unintended contract state
     -   A user is able to update the state of a contract such that it is no longer useable (e.g permanently lock a mutex).
     -   Any assets get unexpectedly "stuck" in a contract with regular use of the contract's public methods.
+    -   An action taken in the staking contracts is applied to an incorrect epoch.
 -   Bypassing time locks
-    -   The `AssetProxyOwner` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
-    -   A user is allowed to bypass the `AssetProxyOwner`.
+    -   The `ZeroExGovernor` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
+    -   A user is allowed to bypass the `ZeroExGovernor`.
+-   Incorrect math calculations
+    -   Overflows or underflow result in unexpected behavior.
+    -   The staking reward payouts are incorrect.
```

Comment:
Maybe this, to weed out submissions that are just slight imprecisions?
```suggestion
    -   The staking reward payouts are (significantly) incorrect.
```

Reply:
Is there a threshold that you would consider significant? It seems like something like 10% would weed out "dust" imprecisions, but could also be too stringent.

Reply:
Not sure if % is the best way to express this (% of what?). From the tests, calculations with an absolute error greater than `1e-12` would surprise me, but that kind of error might not be high wreak any real havoc. Maybe it's best to just leave this subjective.

Reply:
Maybe just "excluding minor rounding errors"? I'd want to know about any small imprecisions that we aren't already aware of. Keep in mind that the bug bounty also pays out based on severity, so no big deal if there are some minor submissions.

Reply:
Actually, we should also keep in mind that these are just examples of things that can qualify for the bounty. Probably best to leave them more open ended with that in mind.

Reply:
It isn't a "rounding" error, though, if, for example, our logarithmic approximation is off.